Hugo development (#18)
Voyant-Photonics/voyant-sdk: README.md

## Licensing

This repository contains example code for interacting with Voyant Photonics, Inc. device drivers.

### License Information

- **Example Code**: All example code in this repository is licensed under the [MIT License](LICENSE).
You are free to use, modify, distribute, and include this example code in both open source and commercial projects.

- **Device Driver**: The provided device driver is proprietary code (**not** covered by the MIT License), but may be called from your application for the evaluation of Voyant sensors.

### Using This Code

Intended applications of this code primarly include:

- Use of examples as starting points for custom applications
- Modify and adapt the examples to suit your needs [THIS AND THE POINT ABOVE ARE IDENTICAL, ALBEIT DIFFERENTLY WORDED. CUT ONE AND REPLACE IF REQUIRED]
- Share your improvements with the community

> **IMPORTANT:** When using the provided example code, retain the copyright notices at the top of each file. Not doing so is a violation of the [MIT License](LICENSE).

Reviewer: Revert this change. IMO this changes tone from polite to aggressive

```suggestion
When using the example code, please retain the copyright notices at the top of each file.
```
